apps/fluent-tester/src/testPages.ts

```diff
@@ -163,13 +163,13 @@ export const tests: TestDescription[] = [
     platforms: ['macos', 'win32'],
   },
   {
-    name: 'MenuButton',
+    name: 'MenuButton Pre-V1',
     component: MenuButtonTest,
     testPage: HOMEPAGE_MENUBUTTON_BUTTON,
     platforms: ['macos', 'win32'],
   },
   {
-    name: 'MenuButton (Experimental)',
+    name: 'MenuButton V1',
     component: ExperimentalMenuButtonTest,
     testPage: HOMEPAGE_EXPERIMENTAL_MENU_BUTTON,
     platforms: ['macos', 'win32'],
```

Comment:
Is MenuButton V1? Or is it going away with MenuV1? I wasn't actually sure. @chiuam fyi

Reply:
MenuButton is its own component, but we should probably redo it because it uses the old ContextualMenu.

Reply:
https://react.fluentui.dev/?path=/docs/components-button-menubutton--default

Reply:
Is it worth calling V1 right now then?

Reply:
I guess not. Idk what to name it instead though, suggestions?

Reply:
.... experimental 😅

One of them is a macOS native module, so there's opportunity to move that to a package and maybe just.. not have the old one? Idk

Reply:
I'm not a huge fan of experimental since it seems to be causing confusion. I think V1 is ok since it's meant to be the test page for the v1 version of the MenuButton, and it will be changed in the future to use Menu. But if you think that will be confusing too I can leave it at experimental

Reply:
I guess "V1" with proper platform status is fine. That's why those are separate =)